fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -582,6 +586,7 @@ struct LogGenerationData : NonCopyable, public ReferenceCounted<LogGenerationDat
 	Version unrecoveredBefore = 1;
 	Version recoveredAt = 1;
 
+	// why do we need it, what does it do?
```

Comment:
this is data center ID, to differentiate primary, satellite, and remote tlogs.

Reply:
do we need it in tlog group or tlog server? here it is tlog group